tests/queries/0_stateless/03164_parts_filter_profiles_test.sql

```diff
@@ -0,0 +1,25 @@
+set log_queries=1;
+set log_queries_min_type='QUERY_FINISH';
+
+DROP TABLE IF EXISTS 03164_query_log_test_parts_filter_profiles;
+CREATE TABLE 03164_query_log_test_parts_filter_profiles
+(
+    a Int64, b Int64, c Int64,
+    INDEX idx_c(c) TYPE minmax
```

Comment:
Is it right that the `minmax` index does not affect the value of the profile events? If yes, then let's remove it because it distracts from the actual purpose of the test.